pip/pep425tags.py

```diff
@@ -158,6 +158,8 @@ def get_supported(versions=None, noarch=False):
 
     abis.append('none')
 
+    arches = []
+
     if not noarch:
```

Comment:
`noarch` is an argument to the function:

```python
def get_supported(versions=None, noarch=False):
```

I could switch the order of the `if` and `else` statements, though, which would make the `if` statement `if noarch`.